packages/amplify-migration/src/app_storage_definition_fetcher.ts

```diff
@@ -21,32 +35,88 @@ export class AppStorageDefinitionFetcher {
     const contents = await fs.readFile(filePath, { encoding: 'utf8' });
     return JSON.parse(contents);
   };
+  private getFunctionPath = (functionName: string) => {
+    return path.join('amplify', 'backend', 'function', functionName, 'src');
+  };
+  private getStorageTriggers = (
+    connections: GetBucketNotificationConfigurationCommandOutput,
+  ): Partial<Record<StorageTriggerEvent, Lambda>> => {
+    const triggers: Partial<Record<StorageTriggerEvent, Lambda>> = {};
+
```

Comment:
nit: store `connections.LambdaFunctionConfiguration` in a variable for repeated access.

Reply:
Addressed. [4559ff3](https://github.com/aws-amplify/amplify-cli/pull/13928/commits/4559ff3b5586e6f88ec42465d36789570e396429)